src/templates/react/Content/.template.config/template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "author": "Maxime Mangel",
+  "classifications": [ "Fable" ],
+  "name": "Simple elmish application using React as a renderer",
```

Comment:
This looks too long on screen, could you please change it to `Fable.Elmish.React App`? Thanks!
<img width="1177" alt="screen shot 2017-04-05 at 17 14 39" src="https://cloud.githubusercontent.com/assets/8275461/24714784/cadfbf64-1a29-11e7-8784-0497634f7403.png">